src/invoice2data/input/pdfplumber.py

```diff
@@ -19,7 +19,8 @@ def to_text(path):
     try:
         import pdfplumber
     except ImportError:
-        logger.debug("Cannot import pdfplumber")
+        logger.error("Cannot import pdfplumber")
+        return "".encode("UTF-8")
```

Comment:
Returning empty string suggests that invoice was parsed but was empty.
If we want to return some value then make it `None` please.

If you take a look at `pdftotext.py` however, you'll see it raises `EnvironmentError` if `pdftotext` is missing. So returning `None` will make `pdfplumber.py` somehow incompatible with the `pdftotext.py`.

As for decision what is better: `return None` or `raise EnvironmentError` - I have no idea or preference.

Reply:
@rmilecki I agree, we need to think of an solution for this.

Returning None conflicts with:
https://github.com/invoice-x/invoice2data/blob/a5bdd50b1cfc1e7953b0d1467c9915bafacbc3aa/src/invoice2data/main.py#L89

as Nonetype cannot be decoded
(Maybe we can remove the decode line? I assume it is a python2 leftover)

pdftotext might be a different story, as it is one of the default/main parsers.
So making everything fail when it's unavailable is not a big dead.

Is there a way to raise the error, only if the pdfplumber input module is called?
We don't want the whole lib to fail on this missing requirement.

This pr https://github.com/invoice-x/invoice2data/pull/491
is currently failing because of the missing pdfplumber.
(the test should not even run when it is unavailable but that's a different sunbject)
In that example, there should be an `ImportError` or `EnvironmentError`